ioctl/util/util.go

```diff
@@ -33,13 +35,23 @@ import (
 	"github.com/iotexproject/iotex-core/ioctl/output"
 	"github.com/iotexproject/iotex-core/ioctl/validator"
 	"github.com/iotexproject/iotex-core/pkg/log"
+
+	"github.com/iotexproject/iotex-proto/golang/iotexapi"
+
+	"github.com/grpc-ecosystem/go-grpc-middleware/util/metautils"
```

Comment:
move these 2 together with `"google.golang.org/grpc/metadata"`

Reply:
Package `iotexapi` is used to create new `APIServiceClient`, and package `metautils` and `metadata` are used to do JWT auth for new context.

Reply:
yes, i mean, move these 2 up to line 24